command/connect/envoy/testdata/defaults.golden

```diff
@@ -14,7 +14,7 @@
     "metadata": {
       "namespace": "default",
       "partition": "default",
-      "envoy_version": "1.18.4"
+      "envoy_version": "1.19.1"
```

Comment:
@eculver maybe afterwards you could knock this one out too: https://github.com/hashicorp/consul/issues/11134